.circleci/config.yml

```diff
@@ -220,55 +247,135 @@ run_tests_from_wheel: &run_tests_from_wheel
 # -------------------------------------------------------------------------------------
 # Jobs
 # -------------------------------------------------------------------------------------
+# ------- CPU only jobs -------
+# Yes, this is horribly redundant. Too lazy to make sense of CircleCI's parameterize 
+# stuff. Might fix when it gets too awful
 jobs:
-  py37_linux:
+  # Pre-commit tests
+  py38_precommit:
+    docker:
+      - image: cimg/python:3.8
+    steps:
+      - checkout
+      - <<: *install_suitesparse
+      - <<: *install_functorch
```

Comment:
do we still need install_functorch since it is now in pytorch?

Reply:
Not sure actually, let me check.

Reply:
Yes, this works. Curiously, we were installing torch via the command to install functorch.